Extensions/ArtifactEngine/Engine/cilogger.ts

```diff
@@ -0,0 +1,21 @@
+var tl = require('vsts-task-lib');
+
+const area: string = 'artifact-engine';
+
+function getDefaultProps() {
+    return {
+        releaseuri: tl.getVariable('Release.ReleaseUri'),
+        releaseid: tl.getVariable('Release.ReleaseId'),
+        builduri: tl.getVariable('Build.BuildUri'),
+        buildid: tl.getVariable('Build.Buildid')
+    };
+}
+
+export function publishEvent(feature, properties: { [key: string]: any }): void {
+    try {
+        tl.assertAgent('2.125.0');
```

Comment:
what happens when agent is less than 2.125.0? will this not log telemetry? or will it fail?